test/parse_request.cc

```diff
@@ -1791,9 +1792,14 @@ TEST(ParseRequest, test_transport_type) {
 
   costing = Costing::bicycle;
   transport_type_key = "bicycle_type";
-  for (const auto& transport_type_value : {"Road", "Cross", "Hybrid", "Mountain"}) {
+  for (const auto& transport_type_value : {"road", "cross", "hybrid", "mountain"}) {
```

Comment:
i would randomly case these

Reply:
Am I not doing this below?
I am executing `test_transport_type_parsing` with upper and lowercased input